src/AutoRest.CSharp/DataPlane/Generation/DataPlaneClientWriter.cs

```diff
@@ -419,6 +423,16 @@ private void WriteStartOperationOperation(CodeWriter writer, LongRunningOperatio
             writer.Line();
         }
 
+        private string? GetXmlDocumentationSummaryText(RestClientMethod restClientMethod)
+        {
+            return restClientMethod.Summary.IsNullOrEmpty() ? restClientMethod.Description : restClientMethod.Summary;
```

Comment:
maybe we just not print the `<Summary>` or `<Remarks>` if the corresponding property is empty?

Reply:
Correct. This logic is handled in `writer.WriteXmlDocumentation(Summary)`. If there is no content for the label, the whole label will not be generated.